x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/__jest__/processors/processor.helpers.tsx

```diff
@@ -103,6 +103,13 @@ const createActions = (testBed: TestBed<TestSubject>) => {
   };
 };
 
+export const commonFormFields = {
```

Comment:
I couldn't find where this object is used. Could you please check if it's still needed?

Reply:
Removed.